index.html

```diff
@@ -64,5 +64,159 @@ <h1 class="headline-alexandra_small">
+        <div class="right">
+          <h1>Tonic</h1>
+          <div class="back">
+            <div>CANOPY</div>
+            <div class="circle"></div>
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+
+      <div class="tonic">
+        <div><img src="images/Availablity.png" alt="" /></div>
+        <div class="right">
+          <h1>
+            Multi-Post<br />
+            Stories
+          </h1>
+          <div class="back">
+            <div>CANOPY</div>
+            <div class="circle"></div>
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+
+      <div class="tonic">
+        <div><img src="images/flex.png" alt="" /></div>
+        <div class="right">
+          <h1>Tonic</h1>
+          <div class="back">
+            <div>CANOPY</div>
+            <div class="circle"></div>
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+
+      <div class="tonic">
+        <div><img src="images/proffesional.png" alt="" /></div>
+        <div class="right">
+          <h1>
+            Multi-Post<br />
+            Stories
+          </h1>
+          <div class="back">
+            <div>CANOPY</div>
+            <div class="circle"></div>
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <h1>About Myself</h1>
+      <p class="description">
```

Comment:
- [ ] the same applies here too, It would be better for this title to use another ```hx```  like ```h2, h3, h4, ...``` to replace the ```<h1>``` as we only have to use a single ```h1``` on the page title to indicate what the entire web page is about 😎😎